### ISSUE TITLE
Missing checkpoint.pth file

### ISSUE THREAD

**Issue by Violet-Evergarde** [hpguo1982/DDANet#1]
Thank you very much for the code, it is very valuable for reference！But there are some questions.
When running test.py, it prompts that the checkpoint.pth file is missing. Can you provide it? Thank you very much!
I would also like to ask if the dataset you provided includes a training set and a validation set? How did you divide them?

**Comment by hpguo1982** [comment]
&nbsp;Dear Violet-Evergarde,



Thank you for your interest in our DDANet project （DDANet: A deep dilated attention network for intracerebral haemorrhage segmentation - Liu - 2024 - IET Systems Biology - Wiley Online Library）.


During our manuscript revision process, the reviewers requested cross-validation to verify model effectiveness.&nbsp;




Consequently, we removed the .pth checkpoint file from the repository.&nbsp;




Regrettably, we've now discovered that the original file was lost during a system reinstallation.&nbsp;




We sincerely apologize for any inconvenience this may cause to your research.

Best Regards,




Huaping Guo

***@***.***&nbsp;

Xinyang Normal University
&nbsp;


------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Wed, Apr 2, 2025 01:39 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; 
Subject: &nbsp;[hpguo1982/DDANet] Missing checkpoint.pth file (Issue #1)

&nbsp;

   
Thank you very much for the code, it is very valuable for reference！But there are some questions.
 When running test.py, it prompts that the checkpoint.pth file is missing. Can you provide it? Thank you very much!
 I would also like to ask if the dataset you provided includes a training set and a validation set? How did you divide them?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;
  Violet-Evergarde created an issue (hpguo1982/DDANet#1)
 
Thank you very much for the code, it is very valuable for reference！But there are some questions.
 When running test.py, it prompts that the checkpoint.pth file is missing. Can you provide it? Thank you very much!
 I would also like to ask if the dataset you provided includes a training set and a validation set? How did you divide them?

**Comment by Violet-Evergarde** [comment]
> &nbsp;亲爱的 Violet-Evergarde，
> 
> 感谢您对我们的 DDANet 项目（DDANet：用于脑出血分割的深度扩张注意力网络 - Liu - 2024 - IET Systems Biology - Wiley Online Library）感兴趣。
> 
> 在我们的手稿修订过程中，审稿人要求进行交叉验证以验证模型的有效性。
> 
> 因此，我们从存储库中删除了 .pth 检查点文件。
> 
> 遗憾的是，我们现在发现原始文件在系统重新安装过程中丢失了。
> 
> 对于由此给您的研究带来的任何不便，我们深表歉意。
> 
> 此致敬意
> 
> 郭华平
> 
> @***.***&nbsp;
> 
> 信阳师范学院
> &nbsp;
> […](#)

Well, I'm sorry, but thank you very much for the code and data, they are still very valuable for reference
